zen/.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/zen.iml" filepath="$PROJECT_DIR$/.idea/zen.iml" />
+    </modules>
+  </component>
+</project>
```

Comment:
this is an IDE file, I would recommend you add `.idea` to your own global gitignore file